presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -485,6 +486,22 @@ public void testBetweenOperatorFilter()
     }
 
     @Test
+    public void testSymbolEqualsSymbolFilter()
+    {
+        Expression symbolEqualsSymbol = new ComparisonExpression(EQUAL, new SymbolReference("x"), new SymbolReference("x"));
```

Comment:
I think we shouldn't have unit test for that, as the (more) correct solution would be to fix the party responsible for introducing/retaining such (almost) useless comparison. We _do_ need an integration test, however, e.g. in `TestTpchLocalStats`